docs/development/backend.md

```diff
@@ -18,33 +18,33 @@ redirects the requests to the defined proxies.
 The backend (Headlamp's server) can be quickly built using:
 
 ```bash
-make backend
+npm run backend
```

Comment:
I think the `run backend`, even though matching what we had is more confusing, since this is not running backend but building it. Maybe `npm run build:backend` and `npm run backend` would actually try to run the backend. WDYT?

Reply:
I will change `npm run backend` so it runs the backend. So it is the same as `npm run backend-run`. Same thing with `npm run frontend`.

I don't think `npm run build:backend` is a good idea to do or discuss now.